fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -464,21 +464,22 @@ protected void updateProcedures() {
 
         // Now only apply the procedures in the model. Much faster than
         // going through the whole schema
-        try {
-            try (Connection c = createConnection()) {
+        try (ITransaction tx = TransactionFactory.openTransaction(connectionPool)) {
```

Comment:
I think its fine, but where were we starting the transaction before?

Reply:
We were not starting a transaction before, we were using a statement which autocommitted the change.

Reply:
ah, ok.  i remember trying something like this in the past and getting into trouble with the migration code...but I think that was before robin cleaned up all the transaction handling.  tests are passing, so i'm good if @punktilious is